sample/app/build.gradle

```diff
@@ -0,0 +1,156 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.0'
+        // To depend on the deployed version of the plugin add the following line here:
+        // classpath 'com.novoda:gradle-android-command-plugin:1.3.0'
```

Comment:
Yes, I thought the purpose of this PR was to actually get rid of this mismatch between intended usage and usage in sample.